docs/source/onedal/algorithms/covariance/covariance.rst

```diff
@@ -37,6 +37,52 @@ Given a dataset :math:`X = \{ x_1, \ldots, x_n \}` with :math:`n` feature vector
 the means is a :math:`1 \times p` matrix, the covariance and the correlation matrices are :math:`p \times p` square matrices.
 The means, the covariance, and the correlation are computed with the following formulas:
 
+.. list-table::
+   :widths: 20 50
+   :header-rows: 1
+   :align: left
+
+   * - Statistic
+     - Definition
+   * - Means
+     - :math:`M = (m_{1}, \ldots , m_{p})`, where :math:`m_{j}=\frac{1}{n}\sum _{i}{x}_{ij}`
+   * - Covariance matrix (sample)
+     - :math:`Cov = (v_{ij})`, where :math:`v_{ij}=\frac{1}{n-1}\sum_{k=1}^{n}(x_{ki}-m_{i})(x_{kj}-m{j})`, :math:`i=\overline{1,p}`, :math:`j=\overline{1,p}`
+   * - Covariance matrix (maximum likelihood)
+     - :math:`Cov' = (v'_{ij})`, where :math:`v'_{ij}=\frac{1}{n}\sum_{k=1}^{n}(x_{ki}-m_{i})(x_{kj}-m{j})`, :math:`i=\overline{1,p}`, :math:`j=\overline{1,p}`
+   * - Correlation matrix
+     - :math:`Cor = (c_{ij})`, where :math:`c_{ij}=\frac{v_{ij}}{\sqrt{v_{ii}\cdot v_{jj}}}`, :math:`i=\overline{1,p}`, :math:`j=\overline{1,p}`
+
+.. _covariance_p_math:
+
+Partial Computing
+-----------------
+
+Given a block of a :math:`X = \{ x_1, \ldots, x_n \}` dataset with :math:`n` feature vectors of :math:`p` dimension,
+the sums is a :math:`1 \times p` matrix, the cross product is :math:`p \times p` square matrices.
+The sums and cross product are computed with the following formulas:
+
+.. list-table::
+   :widths: 20 50
+   :header-rows: 1
+   :align: left
+
+   * - Statistic
+     - Definition
+   * - Sums
+     - :math:`S = (m_{1}, \ldots , m_{p})`, where :math:`m_{j}=sum _{i}{x}_{ij}`
+   * - Cross product matrix
+     - :math:`Crossproduct = (v_{ij})`
```

Comment:
```suggestion
     - :math:`Crossproduct = (v_{ij}) = \sum_{k=1}^{n}(x_{ki}-m_{i})(x_{kj}-m{j})`, :math:`i=\overline{1,p}`, :math:`j=\overline{1,p}`
```

I think also that v_ij needs to be changed to some other name because v_ij is already used in covariance definition.